test/v1/controllers/ListBsasControllerSpec.scala

```diff
@@ -202,6 +204,25 @@ class ListBsasControllerSpec
         status(result) shouldBe OK
         contentAsJson(result) shouldBe summariesJSONWithHateoas(nino)
         header("X-CorrelationId", result) shouldBe Some(correlationId)
+
+        val detail: GenericAuditDetail =
```

Comment:
can we create a common method to create these ** GenericAuditDetail** and ** event** objects